### PR TITLE
FIX Delete Home Button

### PR DESCRIPTION
**Pull request by medBelaid**
Ref(#24)[https://trello.com/c/wlr2O5z6]

### CODE REVIEW

**Comment by achr3f** [comment]
@medBelaid  plz try always to add a short description for you ticket and commit message 

```
Titile

Description ....

REF [#NB]( Link )

Details
-------------------

```

FIX also the branch name  `FIX_logo_home_icon`

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -9,9 +9,18 @@
...
 }
+.logo:hover{
+
+}
+.active_logo{
+
+}
+.inactive_logo{
+
+}
```
Comment (achr3f) [inline app/assets/stylesheets/application.scss:22]: DEAD code

**Comment by achr3f** [comment]
Your Pull Request is not up to date, plz updated it

**Comment by medBelaid** [comment]
understood

2016-01-28 10:46 GMT+01:00 achr3f notifications@github.com:

**Comment by medBelaid** [comment]
done

2016-01-28 10:47 GMT+01:00 mohamed belaid medbbelaid@gmail.com:

**Pull request closed without merging**